package.json

```diff
@@ -186,7 +186,7 @@
     "vscode:package": "npx vsce package",
     "vscode:publish": "npx vsce publish",
     "vscode:prepublish": "npm run build:prod",
-    "build:prod": "webpack --mode production",
+    "build:prod": "cross-env NODE_OPTIONS=--openssl-legacy-provider webpack --mode production",
```

Comment:
That's needed in order to build with the current node 18 (LTS). Using cross-env so that it also works under Windows.